File: baekjoon/01-입출력/js/10950.js
// 백준 제출 시, readFileSync('/dev/stdin') 으로 변경할 것
const fs = require("fs");
const input = fs.readFileSync("예제.txt").toString().split("\n");
const num = parseInt(input[0]);
for (let i = 0; i < num; i++) {
  let row = input[i + 1].split(" ");
  console.log(parseInt(row[0]) + parseInt(row[1]));
}

File: baekjoon/01-입출력/c++/10950.cpp
#include <iostream>

using namespace std;

int main()
{
    int num, i;
    cin >> num;
    int a[num], b[num];
    for(i = 0; i<num; i++){
        cin >> a[i] >> b[i];
    }
    for (i = 0; i<num; i++)
    cout << a[i]+b[i] << endl;
    return 0;
}
